geometry/eolearn/geometry/morphology.py

```diff
@@ -27,33 +23,29 @@ class ErosionTask(EOTask):
     """
 
     def __init__(self, mask_feature, disk_radius=1, erode_labels=None, no_data_label=0):
-        self.mask_type, self.mask_name, self.new_mask_name = next(iter(self._parse_features(mask_feature,
-                                                                                            new_names=True)))
-        self.disk_radius = disk_radius
+        if not isinstance(disk_radius, int) or disk_radius is None or disk_radius < 1:
+            raise ValueError('Disk radius should be an integer larger than 0!')
```

Comment:
What happens now if `disk_radius = 0` ? I don't see this covered. No warning or exception?

Reply:
See the last case (disk_radius < 1) and check test_morphology.py, line 21.

Reply:
Sorry, I'm an idiot.